### ISSUE TITLE
Add new pin command for current version only

### ISSUE THREAD

**Issue by vertigo220** [chocolatey/choco#1997]
As mentioned in #730 (https://github.com/chocolatey/ChocolateyGUI/issues/730#issuecomment-582110166), add a separate pin command for the current new version only, so that it won't update, or ask to update, until the next version. Example commands might be "choco pintemp" or "choco pinver." And once this is done, corresponding entries should be added in the GUI context menu.

A couple additional thoughts to consider:

1) If an app is temporarily pinned, when an auto-update is run, that app should probably not update even if a new version past the ignored one has been released, as the user may wish to skip that one as well. So other, non-pinned, apps should update normally, but that one should show a notification/warning of some sort to check if the user would like to update or re-(temp)pin. This could be an option for global temp pin behavior or could be set individually each time the feature is used by asking the user how to behave when a newer version is released.
2) This is essentially an offshoot of the first point, but it should provide the option to skip minor version updates and only update for major releases.

**Comment by ferventcoder** [comment]
Not sure what exactly this means, but maybe I need to read it a few times.

**Comment by vertigo220** [comment]
Basically, currently there is just the "pin" command, which tells chocolatey to ignore all future updates. What I'm suggesting is one or more new pin-related commands to accomplish any one of the following, depending on what the user wants to do exactly (in addition to the current action of ignoring ALL future updates):

1) Ignore only the current update version, so the next version, no matter what, will trigger the update.
2) Ignore all updates until the next major update, i.e. all minor updates (e.g. current version is 2.1, so 2.2, 2.3, etc will be ignored, and only 3.x will trigger an update). A secondary option for this would be to ignore only minor versions for the current major version (so after updating to 3.0, for example, 3.1 will be done) or for all versions (after updating to 3.0, 3.1 will NOT be done, and it will wait now for 4.x).
3) There should be an option associated with this so that when a version is reached that triggers an update (e.g. 3.0 above), it will either automatically update or ask the user whether to update or re-pin (3.0 is released, user is asked whether to update or to re-ignore until the next minor or major version or indefinitely).

For the GUI, probably the best way to do all of this would be to simply have it pop up a dialog when an app is pinned, asking how long to pin for and what to do once action is needed. So for example, I right-click on "Example App" v1.2 and select "Pin" and it shows a dialog that asks the following:

Pin Duration:
o Indefinitely
o Until next major version
o Until next minor version

Action to take when pin expires:
o Automatically update
o Ask me

Obviously doing this via the CLI would be different, but could probably just be done with the "pin" command and parameters, such as "pin /indef /auto" or "pin /major /ask" but the GUI example should at least help explain what I mean.

Hopefully that helps clear it up. Let me know if I can clarify anything further.

**Comment by TheCakeIsNaOH** [comment]
The feature request to add functionality to pin to skip the next version already exists here at #217

**Comment by TheCakeIsNaOH** [comment]
Duplicate of #217
